### ISSUE TITLE
Incomplete installation message on macOS

### ISSUE THREAD

**Issue by bigardone** [elixir-lsp/elixir-ls#783]
### Environment

* Elixir & Erlang versions (elixir --version): 
```
Erlang/OTP 25 [erts-13.1.2] [source] [64-bit] [smp:10:10] [ds:10:10:10] [async-threads:1] [jit]

Elixir 1.14.2 (compiled with Erlang/OTP 25)
```
* Elixir Language Server version: master [b4a6729]
* Operating system: macOS Ventura 
* Editor or IDE name (e.g. Emacs/VSCode): neoVim
* Editor Plugin/LSP Client name and version: Coc

### Current behavior

After pulling the latest changes from the `master` branch, the language server fails to start due to the following error, whilst if I install back to the previous commit, it works fine:

```
[coc.nvim]: UnhandledRejection: Pending response rejected since connection got disposed
Error: Pending response rejected since connection got disposed
    at Object.dispose (/Users/myuser/.local/share/nvim/site/pack/packer/start/coc.nvim/build/index.js:51:4560)
    at Object.dispose (/Users/myuser/.local/share/nvim/site/pack/packer/start/coc.nvim/build/index.js:239:87474)
    at rd.handleConnectionClosed (/Users/myuser/.local/share/nvim/site/pack/packer/start/coc.nvim/build/index.js:244:10952)
    at rd.handleConnectionClosed (/Users/myuser/.local/share/nvim/site/pack/packer/start/coc.nvim/build/index.js:248:3486)
    at t (/Users/myuser/.local/share/nvim/site/pack/packer/start/coc.nvim/build/index.js:244:10604)
    at gT.invoke (/Users/myuser/.local/share/nvim/site/pack/packer/start/coc.nvim/build/index.js:24:14460)
    at Ru.fire (/Users/myuser/.local/share/nvim/site/pack/packer/start/coc.nvim/build/index.js:24:15230)
    at qe (/Users/myuser/.local/share/nvim/site/pack/packer/start/coc.nvim/build/index.js:25:6504)
    at gT.invoke (/Users/myuser/.local/share/nvim/site/pack/packer/start/coc.nvim/build/index.js:24:14460)
[coc.nvim] Unable to start ElixirLS due to an incomplete erlang installation. See https://github.com/elixir-lsp/elixir-ls/blob/master/guides/incomplete-installation.md
 for guidance.
```

### Expected behavior

The LS should start.

**Comment by AjBreidenbach** [comment]
I was experiencing the same issue with an asdf installation on Ubuntu 20.04.
Switching to tag v0.12.0 resolved the issue for me.
The issue wasn't specific to nvim,  it could also be reproduced by running `./release/language_server.sh`

```
Erlang/OTP 25 [erts-13.1.2] [source] [64-bit] [smp:12:12] [ds:12:12:10] [async-threads:1] [jit:ns]

Elixir 1.14.2 (compiled with Erlang/OTP 25)
```

**Comment by michelboaventura** [comment]
Had the same issue on my Gentoo machine. The problematic commit seems to be https://github.com/elixir-lsp/elixir-ls/commit/b4a672900302b808f75fa46da8f0c07942514884. If I checkout one before it everything works fine.

```
Erlang/OTP 25 [erts-13.1.2] [source] [64-bit] [smp:8:8] [ds:8:8:10] [async-threads:1] [jit:ns]

Elixir 1.14.2 (compiled with Erlang/OTP 25)
```

**Comment by rlopzc** [comment]
Same issue here with commit: 8c34886

I confirm that checking out to 02d3b2e works as before

**Comment by lukaszsamson** [comment]
1. Have you tried installing the deps?
2. Can you make sure you run the server with ‘MIX_ENV=prod’

**Comment by aptinio** [comment]
Deleting the `release` folder fixed it for me! :tada:

**Comment by bigardone** [comment]
Same here! Deleting the folders and running `MIX_ENV=prod mix elixir_ls.release` fixed the issue 🎉

**Comment by jomi-se** [comment]
Same here, using `MIX_ENV=prod` (also figured out the thing about :stream_data so I figured it was the same issue).

I'm not an elixir expert but I ran `mix deps.get` and `mix.deps.compile`. Would those commands be missing something to install the correct stuff?

Maybe it'd be nice to add install steps for newbies building `elixir_ls` from source? (I remember years ago I had to do that for some reason, don't remember exactly why, which is why my current setup didn't use published releases)
